integration_tests/assets/ari_data/mock_ari.py

```diff
@@ -28,7 +28,7 @@
 
 app = Flask(__name__)
 sockets = Sockets(app)
-websocket: WebSocket = None  # type: ignore[assignment]
+websocket: WebSocket = None
```

Comment:
Why remove the type ignore? Doesn't mypy complain that you're assigning `None` to a non-optional type?
Alternatively, annotate with `WebSocket | None`?

Reply:
well, mypy did not complain after removing this, so I dropped it

Reply:
Weird, the annotation is obviously wrong in this case, unless there's a setting to support implicit optional(which is a bad practice and not the case in other wazo projects)

Reply:
Likely mypy is ignoring something because it doesn't understand the type information from the `geventwebsocket.websocket` import(because of the `ignore_missing_import` setting, if there's no type stub for `geventwebsocket.websocket`)

Reply:
OK thanks, I'll revert it back then

Reply:
In general though, what I would prefer for this kind of pattern is to have the runtime code using that variable explicitly acknowledge the expectation that the value is not `None`, by using `assert websocket is not None` for example. This would silence mypy selectively when conscious effort is made by the developer, but mypy would still remind someone who forgot that the variable might be None.

In this case, I don't think anything prevents `websocket` from being None. it's just a matter of implicit API workflow that you're supposed to initiate a websocket connection with `/ari/events` before calling any endpoint that can refer to the global `websocket`. Bad usage of the API can trigger an unexpected error, but since this is mock/test code we don't care too much. If this was production code, I would complain.